libs/feed/website/src/lib/feed-generator.ts

```diff
@@ -48,14 +48,15 @@ export const generateFeed =
       } as Item
     })
 
+    const Feed = (await import('feed')).Feed
```

Comment:
The feed library is quite big and somehow webpack isn't able to properly treshake it out on the client. This fixes it.